twilight-lavalink/src/node.rs

```diff
@@ -107,8 +120,14 @@ impl Error for NodeError {
 pub enum NodeErrorType {
     /// Building the HTTP request to initialize a connection failed.
     BuildingConnectionRequest,
+    /// Sending the HTTP request to Lavalink failed.
+    HttpRequestFailed,
     /// Connecting to the Lavalink server failed after several backoff attempts.
     Connecting,
+    /// You can potentially have no valid session before trying to send outgoing
+    /// events. The session id is obtained in the startup sequence of the node.
+    /// If you attempt to send events before connecting you will error out.
+    OutgoingEventHasNoSession,
```

Comment:
Please reword this to get rid of "you" in the documentation. Describe the error and its causes without addressing the user personally